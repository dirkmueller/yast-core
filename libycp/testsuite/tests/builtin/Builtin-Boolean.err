Parsed:
----------------------------------------------------------------------
"** comparisons **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false == false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true == true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false == true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true == false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false != true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true != false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false != false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true != true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** unary operators **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
true
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
false
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** logical operators **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true && true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true && false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false && true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false && false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true || true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true || false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false || true)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(false || false)
----------------------------------------------------------------------
