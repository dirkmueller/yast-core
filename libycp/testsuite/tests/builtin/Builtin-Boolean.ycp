
# ---------------------------------------------------------
#
#  Filename:	Builtin-Boolean.ycp
#
#  Purpose:	test cases for all boolean builtins
#
# ---------------------------------------------------------


("** comparisons **")

(false == false)
(true == true)
(false == true)
(true == false)

(false != true)
(true != false)
(false != false)
(true != true)


("** unary operators **")

(!false)
(!true)


("** logical operators **")

(true && true)
(true && false)
(false && true)
(false && false)

(true || true)
(true || false)
(false || true)
(false || false)

(nil || true)
(true || nil)
(false || nil)
(nil && true)
(true && nil)
