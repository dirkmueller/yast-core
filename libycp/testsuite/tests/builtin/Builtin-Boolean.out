("** comparisons **")
(true)
(true)
(false)
(false)
(true)
(true)
(false)
(false)
("** unary operators **")
(true)
(false)
("** logical operators **")
(true)
(false)
(false)
(false)
(true)
(true)
(true)
(false)
